docs/99-references/00-references.adoc

```diff
@@ -8,46 +8,49 @@
 
 
 // tag::DE[]
-Dieser Abschnitt enthält Quellenangaben, die ganz oder teilweise im Curriculum referenziert werden.
+Dieser Abschnitt enthält Literaturhinweise, auf deren Basis das Curriculum teilweise entstanden ist.
 // end::DE[]
 
 // tag::EN[]
-This section contains references that are cited in the curriculum.
+This section contains references on which the curriculum is partly based on.
 // end::EN[]
 
 
 A +
 
-[[[aim42+2017]]] Architecture Improvement Method – Open-Source collection of practices and patterns to support software evolution, modernization, maintenance, migration and improvement of software systems. http://aim42.org/[http://aim42.org] as well as http://aim42.github.io/[http://aim42.github.io]
+Architecture Improvement Method – Open-Source collection of practices and patterns to support software evolution, modernization, maintenance, migration and improvement of software systems. https://aim42.org/[https://aim42.org] as well as https://aim42.github.io/[https://aim42.github.io]
```

Comment:
Close, please (re-)add labels and description as well as "formatting" of the characters (`**A**`). See [here for details](https://raw.githubusercontent.com/isaqb-org/advanced-template/master/docs/99-references/00-references.adoc). 

If you apply the template's format, it will look exactly as all other modules. And, if we need the possibility to reference the books, we can easily add that to each chapter.